godot-core/src/classes/class_runtime.rs

```diff
@@ -60,11 +60,7 @@ pub(crate) fn construct_engine_object<T>() -> Gd<T>
 where
     T: GodotClass + Bounds<Declarer = bounds::DeclEngine>,
 {
-    // SAFETY: adhere to Godot API; valid class name and returned pointer is an object.
-    unsafe {
-        let object_ptr = sys::interface_fn!(classdb_construct_object)(T::class_name().string_sys());
-        Gd::from_obj_sys(object_ptr)
-    }
+    Gd::new_alloc_postinited()
```

Comment:
As far as I can tell, this happens for both manual + ref-counted objects; bounds only restrict to engine classes. As such, the name `new_alloc` is misleading.
```suggestion
    Gd::create_with_postinit()
```